BeatsIOSUITests/BeatsIOSUITests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  BeatsIOSUITests.swift
+//  BeatsIOSUITests
+//
+//  Created by Aloisio Formento Junior on 17/03/22.
+//
+
+import XCTest
+
+class BeatsIOSUITests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
```

Comment:
Se não for fazer testes, melhor tirar esse target para não poluir o PR

Reply:
Feito